libs/wingsdk/src/target-sim/api.ts

```diff
@@ -18,6 +15,10 @@ import * as core from "../core";
 export class Api extends cloud.Api implements ISimulatorResource {
   private _routes: ApiSchema["props"]["routes"] = [];
 
+  public get url(): string {
```

Comment:
@Chriscbr 

Can we make sure that the console is able to access this field somehow? (maybe it is already possible, but I am not sure) 

AFAIK, the console (@polamoros ) is using an internal detail of the sim implementation in order to get the url value. The console should only use the inflight api of a resource, because we want to be able to use the console for the tf-aws (and friends) target as well 

 @ainvoner &  @eladb  - FYI

Reply:
@ekeren It should be possible to access the URL through the simulator's public methods:

```js
const sdk = require("@winglang/sdk");
const simulator = new sdk.testing.Simulator({ simfile : "./target/api_simple.wsim"});
await simulator.start();

const queue = simulator.getResourceConfig("root/cloud.Api");
const routes = queue.props.routes;
console.log("routes are:", JSON.stringify(routes));
const url = queue.attrs.url;
console.log("url is:", url);
```

For now I think it is OK to treat resource config as a public API (breaking changes will be communicated with console team and should require semver bumps), as long as it's being accessed through the SDK's classes/methods. cc @ainvoner @polamoros